rt/rs/security/http-signature/src/main/java/org/apache/cxf/rs/security/httpsignature/MessageSigner.java

```diff
@@ -0,0 +1,74 @@
+package org.apache.cxf.rs.security.httpsignature;
+
+import org.tomitribe.auth.signatures.Signature;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class MessageSigner {
+    private final String digestAlgorithmName;
+    private final String signatureAlgorithmName;
+
+    /**
+     * Message signer using standard digest and signing algorithm
+     */
+    public MessageSigner() {
+        this("rsa-sha256", "SHA-256");
```

Comment:
Should `rsa-sha256` and `SHA-256` be encapsulated as constants? I see some of them several times in the code.